shared/p2p/peers_view.go

```diff
@@ -0,0 +1,101 @@
+package p2p
+
+import (
+	"github.com/pokt-network/pocket/logger"
+	"github.com/pokt-network/pocket/shared/crypto"
+	"sort"
+)
+
+type PeersView interface {
+	GetAddrs() []string
+	GetPeers() PeerList
+}
+
+type SortedPeersView struct {
```

Comment:
Should we have a cast `var _ PeersView = &SortedPeerView{}` to enforce the interface at complication time?

Reply:
Yes indeed, thanks for calling this out! :100: :raised_hands: